### PR TITLE
Define Emotional Intelligence at thoughtbot

### PR DESCRIPTION
**Pull request by stratigos**
This is a first draft of _Defining Emotional Intelligence at thoughtbot_, as proposed by the 2019 EQ Task Force. :robot: :heart: 

As for its location within _Guides_, we believe there is an opportunity for thoughtbot to take the lead on the process of "Defining EQ in the workplace". _Guides_ are already a public place where people can learn about how we work. The document can be continuously improved by diverse influencers on GitHub, and being in a public place allows other users, or even companies, to fork our document and contribute. Having this document in a public location provides an opportunity to positively influence the world outside of our company.

### CODE REVIEW

File: working-together/README.md [hunk 1]
```diff
@@ -2,64 +2,6 @@
...
-"My feedback for that is..."
-* **Speak as I:** Try to speak from the *I* perspective. Don't speak for others.
-* **Disagree:** You don't have to agree with everybody, but be aware of how you 
-frame your feedback. Criticize the idea, not the person.
-* **Everyone can participate:** Assuming someone doesn't want to participate 
-can be exclusionary. Give people the opportunity to decline an invitation.
-* **Speak up:** Follow up if team members treat others disrespectfully. If you 
-don't feel comfortable addressing someone directly, speak with your manager or Chief
-People Officer.
+* [Emotional Intelligence](emotional-intelligence.md)
```
Comment (baccigalupi) [inline working-together/README.md]: I am wondering if a title and related page that is more focused on inclusive culture fits better with the section on "Working Together"?

I love the writing and ideas. I am thinking it is broader than just working together though! And for working together, the important part of EQ is not just having it, but its outcomes: better decision making through a safer, happier and more inclusive work environment.

Comment (stratigos) [reply working-together/README.md]: Thanks Kane, I think that suggestion does hold this in a stronger light.

I had an intuition that the _Define Emotional Intelligence_ document was part of a greater "How to: Culture" guide, and none of the current top level guides seemed like the right fit. _Working Together_ was the closest match, though I dont think its the right fit either. 

Ill create a new top level document with a loosely held name of _Inclusive Culture_, and link to the _Define Emotional Intelligence_ from there. Other D&I task forces can build upon _Inclusive Culture_, beyond the EQ part.

**Comment by samkap** [comment]
This looks really great, @stratigos !

File: working-together/README.md (hunk 1 above)
Comment (stratigos) [reply working-together/README.md]: As of ebfd244f23e1489e6a8307c5dff8456964bd48d0 I had moved these documents to their own location in guides, and restored _Working Together_ to its prior state. :truck:

**Pull request merged**